Automattic/jetpack
# Boost: Fix update score incompatibility

#### Changes proposed in this Pull Request:
During updates from 1.2 to 1.3 users would face timeout occasionally. This PR accounts for the issue.

#### Jetpack product discussion
N/A

#### Does this pull request change what data or activity we track or use?
no

#### Testing instructions:
* On a fresh site, install boost 1.2
* Fetch speed scores
* Within 300 seconds(5 minutes), update to boost 1.3 and go to boost settings page.
* The score should try to refetch and work correctly(previously here it would fail).
